SimonOkello/portfolio
# Initial Commit

**What does this PR do?**
Creates project directory structure

**What tasks are to be completed in this PR?**
Create `UI` directory
Add `images` directory into `UI` directory
Add `css` directory into `UI` directory
Add `index.html` into `UI` directory

**How can this be manually tested?**
Fork this repo `https://github.com/SimonOkello/portfolio.git`.
Cd  into  `portfolio` directory. `cd portfolio`.
Checkout into gh-pages branch. `git checkout gh-pages`.
See the project directory structure.

**Screenshot**
![image](https://user-images.githubusercontent.com/28682748/53460544-86816d80-3a4e-11e9-9fc3-8ed62b2b6c13.png)
